source/administration/changelog.md

```diff
@@ -2,7 +2,7 @@
 
 This changelog summarizes updates to [Mattermost Team Edition](http://www.mattermost.org/), an open source team messaging solution released bi-monthly under an MIT license, and [Mattermost Enterprise Edition](https://about.mattermost.com/pricing/), a commercial upgrade offering enterprise messaging for large organizations.
 
```

Comment:
It's bi-monthly still "officially".

I think we'll do an official announce of us moving back to monthly releases next week, after which we should update all of our docs + webpages